### PR TITLE
Bug Fixes and Responsiveness Added

### PR DESCRIPTION
**Pull request by UzeeChamp1**
Minor Bug Fixes
Responsiveness Added for smaller screens
Files are being formatted
Footer-Style has been updated using useStyles

### CODE REVIEW

File: src/Copyright.tsx [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+      sx={{
+        backgroundColor: 'white',
+        paddingBlock: '0.25rem',
+        position: 'absolute',
+        left: 0,
+        bottom: 0,
+        width: '100%'
+      }}
+    >
+      <Typography variant="body2" sx={{ color: '#343E5F' }} align="center">
```
Comment (alitjanjua) [inline src/Copyright.tsx:16]: @UzeeChamp1 can we add this kind of colour code to our mui theme and then use here?

File: src/components/home/services/services.tsx [hunk 2]
```diff
@@ -39,11 +56,11 @@ const ServicesBox: React.FC<ServicesBoxProps> = ({
 
   return (
     <Paper elevation={0}>
-      <Grid container>
-        <Grid item xs={12} sm={3}>
+      <Grid container className={classes.gridRoot}>
```
Comment (alitjanjua) [inline src/components/home/services/services.tsx:59]: naming convention should be really relevant
```suggestion
      <Grid container className={classes.servicesGridRoot}>
```

File: src/components/layout/Footer.tsx [hunk 3]
```diff
@@ -28,20 +19,24 @@ import { RatingStarsIcon } from 'src/components/svg/rating-stars-icon';
...
 import { ClutchIcon } from 'src/components/svg/clutch-icon';
+import useStyles from 'public/assets/styles/footer-style';
+import Copyright from 'src/Copyright';
 
 const Footer = () => {
+  const classes = useStyles();
+  
   return (
-    <Box component="div" style={ContainerStyle}>
+    <Box component="div" className={classes.ContainerStyle}>
```
Comment (alitjanjua) [inline src/components/layout/Footer.tsx:29]:
```suggestion
    <Box component="div" className={classes.footerRoot>
```

**Review by alitjanjua — CHANGES_REQUESTED** [review]
@UzeeChamp1 things are looking great, only thing I find incorrect is naming convention of style classes.

File: src/Copyright.tsx (hunk 1 above)
Comment (UzeeChamp1) [reply src/Copyright.tsx:16]: colors are inconsistent for every heading, that's why I didn't add it in the theme. Even this color code is quite similar to blackish,

**Pull request merged**